### PR TITLE
building images, issue2

### LINKED ISSUE #2: create thumbnails for building themes
URL: https://www.redesigner.io/building

image

Currently building themes are represented by text.

We need to create images to represent each theme.

- [ ] pick image of a house from [unsplash](https://unsplash.com/s/photos/house?license=free). Make sure the image is free for use by choosing free license filter
- [ ] use our app to redesign the same house in all different themes.
- [ ] check if result is clean. If not, try another image from [unsplash](https://unsplash.com/)
- [ ] name all images in the format `${building}-${theme}.jpg` ex: house-rustic.jpg, house-minimalist.jpg and so on
- [ ] And then use some image editing tool to resize al images to 100 x 100 size and then upload them to /public folder

### PR DESCRIPTION
**Pull request by GrumpyArdias**
square images, 100x100

fixes #2

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -33,3 +33,6 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+
+
+.env.local
```
Comment (kirandash) [inline .gitignore:38]: This is not required since .env*.local is already added to .gitignore file.

Please remove this change

**Review by kirandash — CHANGES_REQUESTED** [review]
The images look great. Please undo the change for .gitignore. It's unnecessary

File: .gitignore (hunk 1 above)
Comment (GrumpyArdias) [reply .gitignore:38]: If i do this my .evn.local will be updated to the repo

Comment (kirandash) [reply .gitignore:38]: > If i do this my .evn.local will be updated to the repo

no it should not. gitignore already has .env.local ignored

image

is it showing the file in your git status?

Comment (GrumpyArdias) [reply .gitignore]: I fixt it!

**Review by kirandash — APPROVED** [review]
Perfect! Thanks!

**Pull request merged**